app-emulation/spice-protocol/spice-protocol-0.14.4.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Headers defining the SPICE protocol"
+HOMEPAGE="https://www.spice-space.org/"
+SRC_URI="https://www.spice-space.org/download/releases/${P}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm arm64 ~ia64 ~ppc ppc64 ~riscv ~sparc x86"
```

Comment:
When bumping, we need to reset keywords to mark it as "unstable": `ekeyword \~all foo.ebuild` ([devmanual](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades))